src/rollup/plugins/externals.ts

```diff
@@ -138,6 +150,7 @@ export function externals(opts: NodeExternalsOptions): Plugin {
         if (!isAbsolute(originalId)) {
           const fullPath = await _resolve(originalId);
           trackedExternals.add(fullPath);
+          console.log("[externals]", id);
```

Comment:
I'm sure you're aware, but as it isn't a draft, just a quick comment to mention these three console logs...

Reply:
nice catch btw